test/unit/common/domain/handleCommandTests.ts

```diff
@@ -128,6 +128,61 @@ suite('handleCommand', (): void => {
         await domainEventStore.getLastDomainEvent({ aggregateIdentifier })
       ).is.undefined();
     });
+
+    test(`passes the correct state to the authorize handler.`, async (): Promise<void> => {
```

Comment:
This should be a single-quoted string, as we don't have any quotes inside of it.

```suggestion
    test('passes the correct state to the authorize handler.', async (): Promise<void> => {
```

Reply:
This should be fixed by now.